patches/minecraft/net/minecraft/block/RedstoneWireBlock.java.patch

```diff
@@ -1,5 +1,14 @@
 --- a/net/minecraft/block/RedstoneWireBlock.java
 +++ b/net/minecraft/block/RedstoneWireBlock.java
+@@ -47,7 +47,7 @@
+    private final Map<BlockState, VoxelShape> field_235541_j_ = Maps.newHashMap();
+    private static final Vector3f[] field_235542_k_ = new Vector3f[16];
+    private final BlockState field_235543_o_;
+-   private boolean field_150181_a = true;
++   private static boolean field_150181_a = true;
```

Comment:
Why is this being changed to static? It strikes me like this could quite easily break a bunch of things.

Reply:
Because Redstone wires check if there are powered by disabling themselves (through this field) and looking for other sources. If this is per instance (non static), subclasses would not know when they should disable powering, which results in infinite powering. To prevent this I've changed it to static, so subclasses also disable powering.